server/etc/httpd/conf.d/pulp_content.conf

```diff
@@ -0,0 +1,22 @@
+
+WSGISocketPrefix run/wsgi
+WSGIProcessGroup pulp-content
+WSGIApplicationGroup pulp-content
+WSGIScriptAlias /pulp/content /srv/pulp/content.wsgi
+WSGIDaemonProcess pulp-content user=apache group=apache processes=3 display-name=%{GROUP}
+WSGIImportScript /srv/pulp/content.wsgi process-group=pulp-content application-group=pulp-content
+
+<Files content.wsgi>
+    WSGIPassAuthorization On
+    WSGIProcessGroup pulp-content
+    WSGIApplicationGroup pulp-content
+    SSLRenegBufferSize  1048576
+    SSLVerifyDepth 9
```

Comment:
I'd be more than happy to modify all of them, since they're often set to very odd values (like 2 or 3), but I think maybe a separate PR to fix up our Apache config would be better. I know there is an issue about this.